studio/components/layouts/LogsLayout/LogsMenu.utils.ts

```diff
@@ -0,0 +1,76 @@
+import { Project } from 'types'
+import { IS_PLATFORM } from 'lib/constants'
+import { ProductMenuGroup } from 'components/ui/ProductMenu/ProductMenu.types'
+import { useFlag } from 'hooks'
+
+export const generateLogsMenu = (project?: Project): ProductMenuGroup[] => {
```

Comment:
just to check - the entire logs route is hidden behind IS_PLATFORM right? based on NavigationBar.utils?

in that case that i guess we don't have to put IS_PLATFORM here at all then

Reply:
Yes! You're right actually. Good catch.